supplementary_style_guide/glossary_terms_conventions/general_conventions/q.adoc

```diff
@@ -44,7 +44,7 @@
 
 [[qeth]]
 ==== image:images/yes.png[yes] qeth (adjective)
-*Description*: Messages with a prefix `qeth` are issued by the `qeth` device driver. The `qet`h device driver supports a multitude of network connections, for example, connections through Open Systems Adapters (OSA), HiperSockets™, guest LANs, and virtual switches.
+*Description*: Messages with a prefix `qeth` are issued by the `qeth` device driver. The `qeth` device driver supports a multitude of network connections, for example, connections through Open Systems Adapters (OSA), HiperSockets™, guest LANs, and virtual switches.
```

Comment:
```suggestion
*Description*: Messages with a prefix `qeth` are issued by the `qeth` device driver. The `qeth` device driver supports a multitude of network connections, such as connections through Open Systems Adapters (OSA), HiperSockets™, guest LANs, and virtual switches.
```

This removed the non-restrictive clause (for example) that comes between the connections-connections clauses.

Reply:
Per ISG (https://www.ibm.com/docs/en/ibm-style?topic=format-examples#inline-examples), this "for example" is used properly. 

Either way, this is just fixing a typo, so I think it's fine to merge this change as is and not worry about other fixes in the existing text.